extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusAst.js

```diff
@@ -0,0 +1,68 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusAst',[
+	'ext.wikia.adEngine.context.slotsContext',
+	'wikia.geo',
+	'wikia.instantGlobals',
+	'wikia.location'
+], function (slotsContext, geo, instantGlobals, loc) {
+	'use strict';
+
+	var bidderName = 'appnexusAst',
+		debugPlacementId = '5768085',
+		slots = {
+			oasis: {
+				TOP_LEADERBOARD: {
+					placementId: '5768085' // FIXME
+				},
+				INCONTENT_PLAYER: {
+					placementId: '5768085' // FIXME
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1134)